### ISSUE TITLE
[4.0] Smart Search error on PostgreSQL

### ISSUE THREAD

**Issue by SharkyKZ** [joomla/joomla-cms#29165]
### Steps to reproduce the issue

Use PostgreSQL.
Perform a search in frontend.

### Expected result

Works.

### Actual result

```
22P02, 7, ERROR: invalid input syntax for type bytea LINE 3: ('blog','O:51:"Joomla\\Component\\Finder\\Administrator\\Ind... ^

() 	JROOT\libraries\vendor\joomla\database\src\Pdo\PdoDriver.php:497
2 	Joomla\Database\Pdo\PdoDriver->execute() 	JROOT\libraries\vendor\joomla\database\src\Pgsql\PgsqlDriver.php:790
3 	Joomla\Database\Pgsql\PgsqlDriver->insertObject() 	JROOT\components\com_finder\src\Helper\FinderHelper.php:86
4 	Joomla\Component\Finder\Site\Helper\FinderHelper::logSearch() 	JROOT\components\com_finder\src\View\Search\HtmlView.php:181
5 	Joomla\Component\Finder\Site\View\Search\HtmlView->display() 	JROOT\libraries\src\MVC\Controller\BaseController.php:691
6 	Joomla\CMS\MVC\Controller\BaseController->display() 	JROOT\components\com_finder\src\Controller\DisplayController.php:59
7 	Joomla\Component\Finder\Site\Controller\DisplayController->display() 	JROOT\libraries\src\MVC\Controller\BaseController.php:729
8 	Joomla\CMS\MVC\Controller\BaseController->execute() 	JROOT\libraries\src\Dispatcher\ComponentDispatcher.php:146
9 	Joomla\CMS\Dispatcher\ComponentDispatcher->dispatch() 	JROOT\libraries\src\Component\ComponentHelper.php:384
10 	Joomla\CMS\Component\ComponentHelper::renderComponent() 	JROOT\libraries\src\Application\SiteApplication.php:196
11 	Joomla\CMS\Application\SiteApplication->dispatch() 	JROOT\libraries\src\Application\SiteApplication.php:235
12 	Joomla\CMS\Application\SiteApplication->doExecute() 	JROOT\libraries\src\Application\CMSApplication.php:231
13 	Joomla\CMS\Application\CMSApplication->execute() 	JROOT\includes\app.php:63
14 	require_once() 	JROOT\index.php:36 
```

### System information (as much as possible)

PostgreSQL 11.5

### Additional comments

Coming from this line:

https://github.com/joomla/joomla-cms/blob/65abc0392008982cc33d765e1e1ee4aa3a787bb1/components/com_finder/src/Helper/FinderHelper.php#L84

**Comment by Hackwar** [comment]
This will be fixed by #29428.

**Comment by SharkyKZ** [comment]
Closing as #29428 fixes this.
